templates/secret-redis-preinstall.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if not .Values.redis.enabled }}
+{{- if and (not .Values.redis.auth.existingSecret) (not .Values.redis.existingSecret) }}
```

Comment:
Same, those could all be combined in one `and`?

Reply:
It could. I will simplify this in the other secret as well